types/serverless/classes/Plugin.d.ts

```diff
@@ -56,7 +56,7 @@ declare namespace Plugin {
             };
     }
 
-    type ConfigurationVariablesSource = (variableSource: any) => Promise<any>;
+    type ConfigurationVariablesSource = (variableSource: any) => any;
```

Comment:
let's still include `Promise` in the signature, to indicate that we know it is likely that it could be a promise and thus it's actually worth doing an `await`:

```suggestion
    type ConfigurationVariablesSource = (variableSource: any) => Promise<any> | any;
```

Reply:
could you apply this with an `eslint-disable` comment? I'm 98% sure having this marked as a `Promise` will have tools like `@typescript-eslint` behave slightly different than if it's just an `any`, which is why this is important